backends/dpdk/dpdkHelpers.h

```diff
@@ -178,6 +179,8 @@ class ConvertStatementToDpdk : public Inspector {
     cstring append_parser_name(const IR::P4Parser* p, cstring);
     void set_parser(const IR::P4Parser* p) { parser = p; }
     bool handleConstSwitch(const IR::SwitchStatement* a);
+    void processHashParams(const IR::Argument* field,
+                           IR::Vector<IR::Expression>& components);
```

Comment:
functions not used outside class shall be declared private.

Reply:
Addressed